test/test-grab-project.js

```diff
@@ -116,7 +116,7 @@ test('grab-project: module does not exist', function (t) {
     options: {}
   };
   grabProject(context, function (err) {
-    t.equals(err && err.message, 'Failure getting project from npm');
+    t.equals(err && err.message, 'Failure getting tarball');
```

Comment:
I changed the error message earlier in the commit. Two messages got rolled into one due to coverage, so this message was more suiting